Mlem/Views/Tabs/Inbox/InboxView+Feeds.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  InboxView+Feeds.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-04-01.
+//
+
+import Foundation
+import SwiftUI
+
+extension InboxView {
+    @ViewBuilder
+    var personalFeedView: some View {
+        Section {
+            switch selectedPersonalTab {
+            case .all:
+                InboxFeedView(tracker: personalInboxTracker)
+            case .replies:
+                InboxFeedView(tracker: replyTracker)
+            case .mentions:
+                InboxFeedView(tracker: mentionTracker)
+            case .messages:
+                InboxFeedView(tracker: messageTracker)
+            default:
+                InboxFeedView(tracker: personalInboxTracker)
+                    .onAppear {
+                        assertionFailure("personalFeedView rendered with non-personal tab!")
+                    }
+            }
+        } header: {
+            BubblePicker(InboxTab.personalCases, selected: $selectedPersonalTab, withDividers: [.bottom]) { tab in
+                Text(tab.label)
+            }
+            .background(Color.systemBackground.opacity(scrollToTopAppeared ? 1 : 0))
+            .background(.bar)
+            .animation(.easeOut(duration: 0.2), value: scrollToTopAppeared)
```

Comment:
This code is duplicated several times and would ideally be part of a `picker(tabs: [InboxTab], selected: Binding<InboxTab>)` `@ViewBuilder` method